extra-utils/genie/autobuild/build

```diff
@@ -0,0 +1,4 @@
+cd "$SRCDIR"/genie
```

Comment:
Move this to `spec` as `SUBDIR=genie/genie`.

Reply:
Done